src/main/java/com/hermesworld/ais/galapagos/adminjobs/impl/CleanupDeveloperCertificatesJob.java

```diff
@@ -0,0 +1,56 @@
+package com.hermesworld.ais.galapagos.adminjobs.impl;
+
+import com.hermesworld.ais.galapagos.adminjobs.AdminJob;
+import com.hermesworld.ais.galapagos.devcerts.DeveloperCertificateService;
+import com.hermesworld.ais.galapagos.kafka.KafkaClusters;
+import com.hermesworld.ais.galapagos.kafka.config.KafkaEnvironmentConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CleanupDeveloperCertificatesJob implements AdminJob {
+
+    private final KafkaClusters kafkaClusters;
+
+    private final DeveloperCertificateService developerCertificateService;
+
+    @Autowired
+    public CleanupDeveloperCertificatesJob(KafkaClusters kafkaClusters,
+            DeveloperCertificateService developerCertificateService) {
+        this.kafkaClusters = kafkaClusters;
+        this.developerCertificateService = developerCertificateService;
+    }
+
+    @Override
+    public String getJobName() {
+        return "cleanup-developer-certificates";
+    }
+
+    @Override
+    public void run(ApplicationArguments allArguments) throws Exception {
+        System.out.println();
+        System.out.println(
+                "========================= Starting Cleanup of expired Developer Certificates on all Kafka clusters ========================");
+        System.out.println();
+
+        kafkaClusters.getEnvironments().forEach(cluster -> {
```

Comment:
This loop will fail if **one** of the clusters is not certificate-based. But it is perfectly valid to have e.g. one certificate- and one Confluent-based cluster...
Recommendation: Just leave the whole loop and check out here. And the implementation in DeveloperCertificateService should only perform the cleanup on certificate-based clusters.